NRediSearch/Query.cs

```diff
@@ -205,6 +215,13 @@ internal void SerializeRedisArgs(List<object> args)
                 args.AddRange(_fields);
             }
 
+            if(_sortBy != null)
+            {
+                args.Add("SORTBY".Literal());
+                args.Add(_sortBy.Literal());
```

Comment:
what is `_sortBy` here? if this is user data related (rather than a fundamental part of the API syntax), it should **not** be `.Literal()`; the point of `.Literal()` is to avoid allocating for *repeated* content by caching and reusing an encoded version - but for user data, this is a bad thing and could lead to a slow memory leak